lib/python/rose/config_editor/valuewidget/array/logical.py

```diff
@@ -179,8 +184,25 @@ def populate_table(self):
             self.del_button.hide()
         else:
             self.del_button.show()
+        if self.has_titles:
+            for c, label in enumerate(self.metadata['element-titles']):
+                if c >= len(table_widgets):
```

Comment:
Why does this one not have a minus 1 like the other ones (`if c >= len(table_widgets) - 1:`)?